components/CollectionForm.js

```diff
@@ -95,15 +160,7 @@ export default function CollectionForm({ onSubmitCollection, prevValues }) {
                 onClick={() => setCollectionColor(color)}
               />
             ))}
-            <input
-              type="color"
-              name="color"
-              id="color"
-              value={collectionColor}
-              onChange={(event) => setCollectionColor(event.target.value)}
-              required
-            />
-          </StyledColorPicker>
+          </StyledColorsContainer>
 
           <StyledButtonContainer>
             <Button type="submit" onClick={handleCancel}>
```

Comment:
`type="submit"` should be set for the button submitting the form, not the one canceling.

```suggestion
            <Button type="button" onClick={handleCancel}>
            Reset
            </Button>
            <Button type="submit" buttonVariant="create">
              {prevValues?.id ? "Update" : "Create"}
</Button>
```